apps/docs/src/app/platform/component-docs/locale/messages.xlf

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Why do you have XLS files here? I think we said you will remove them ? First we need to have end2end process with schematics that cna add these files to the project. As you know this would not work right ?


SourceFile path have relative path to the file, but on the application level you showed me that path starts with ../nodemodules/..

Reply:
regarding i18n, when I checked about the path issue you were mentioning
![image](https://user-images.githubusercontent.com/53509521/85704972-717a2480-b6fe-11ea-981c-5f6d88b6c416.png)
the one that starts with `../../libs` is the static string from our library, but the one starting with `src/app/` are the i18n markers placed by other components in the documentation pages. 
Until we bring in the schematics, when an application using ngx runs `ng xi18n`, it will not consider this `messages.xlf` at all, it will generate its own `messages.xlf` where the path will anyway resolve to `../../nodemodules`. It will work fine for them.
We are keeping i18n and` messages.xlf `here, because as discussed, we are not working on app-framework(demo app) for now, so we would be bringing i18n also to ngx for Puru to test the whole loop.

This is why I kept those files and the ICU expression logic for counter message.